src/components/Warnings/StaleDataWarnings.js

```diff
@@ -1,44 +1,25 @@
-import { connect } from 'react-redux';
-import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
 import React from 'react';
 import Warning from './Warning';
+import {
+  selectAggsHasDataIssue,
+  selectAggsIsDataStale,
+} from '../../reducers/aggs/selectors';
 
-const WARN_DATA_ISSUE =
+export const WARN_DATA_ISSUE =
   'We’re currently experiencing technical issues that' +
   ' have delayed the refresh of data on the Consumer Complaint Database.  We' +
   ' expect to refresh the data in the next few days.';
 
-const WARN_DATA_STALE =
-  'We’re currently experiencing technical issues that' +
-  ' have delayed the refresh of data in the ' +
-  'Consumer Complaint Database.  We expect to refresh the data in the next ' +
-  'few days.';
-
-export class StaleDataWarnings extends React.Component {
-  // eslint-disable-next-line complexity
-  render() {
-    const { hasDataIssue, isDataStale } = this.props;
-    const anything = hasDataIssue || isDataStale;
-
-    if (!anything) return null;
-
-    return (
-      <div>
-        {hasDataIssue && <Warning text={WARN_DATA_ISSUE} />}
-        {isDataStale && <Warning text={WARN_DATA_STALE} />}
-      </div>
-    );
-  }
-}
-
-const mapStateToProps = (state) => ({
-  hasDataIssue: state.aggs.hasDataIssue,
-  isDataStale: state.aggs.isDataStale,
-});
-
-export default connect(mapStateToProps)(StaleDataWarnings);
-
-StaleDataWarnings.propTypes = {
-  hasDataIssue: PropTypes.bool,
-  isDataStale: PropTypes.bool,
+export const StaleDataWarnings = () => {
+  const hasDataIssue = useSelector(selectAggsHasDataIssue);
+  const isDataStale = useSelector(selectAggsIsDataStale);
+  const hasError = hasDataIssue || isDataStale;
```

Comment:
This is such a small component that it probably doesn't matter, but maybe use `useMemo` here?

Reply:
I think we should only use useMemo when there's computationally expensive calculations that occur. 

https://kentcdodds.com/blog/usememo-and-usecallback#how-is-usememo-different-but-similar